src/traces/histogram/attributes.js

```diff
@@ -71,6 +71,16 @@ module.exports = {
         ].join(' ')
     },
 
+    mode: {
```

Comment:
or maybe alternatively, a boolean `cumulative` attribute could do the trick. I don't see any other possible _modes_ for histograms.